### PR TITLE
Canada main federal tax rates

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Canada federal income tax Issue #4

### CODE REVIEW

File: policyengine_canada/parameters/gov/income_tax_schedule.yaml [hunk 1]
```diff
@@ -1,22 +1,41 @@
-description: Example marginal tax schedule.
+description: Canada federal tax rates
 brackets:
   - threshold:
       values:
-        2010-01-01: 10_000
+        2010-01-01: 0
```
Comment (nikhilwoodruff) [inline policyengine_canada/parameters/gov/income_tax_schedule.yaml:5]: Could we date these from 2021? Looks like they weren't in effect before.

File: policyengine_canada/parameters/gov/age_exemption.yaml [hunk 2]
```diff
@@ -6,5 +6,5 @@ metadata:
   name: income_tax_max_age
   label: Income tax age exemption
   reference:
-  - title: Title
-    href: https://example.com
\ No newline at end of file
+  - title: Government of Canada - 9.2.4 Tax brackets and rates
+    href: https://www.canada.ca/en/financial-consumer-agency/services/financial-toolkit/taxes-quebec/taxes-quebec-2/5.html
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/age_exemption.yaml:10]: I don't see this specified here

File: policyengine_canada/parameters/gov/income_tax_schedule.yaml [hunk 3]
```diff
@@ -1,22 +1,41 @@
-description: Example marginal tax schedule.
+description: Canada federal tax rates
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/income_tax_schedule.yaml:1]: Move to `parameters/gov/cra/tax/income/rates.yaml`

File: policyengine_canada/parameters/gov/age_exemption.yaml [hunk 4]
```diff
@@ -6,5 +6,5 @@ metadata:
   name: income_tax_max_age
   label: Income tax age exemption
   reference:
-  - title: Title
-    href: https://example.com
\ No newline at end of file
+  - title: Government of Canada - 9.2.4 Tax brackets and rates
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/age_exemption.yaml:9]: Move to `parameters/gov/cra/tax/income/exempt/age.yaml`

Also specify if this is inclusive of 65 or not

File: policyengine_canada/parameters/gov/age_exemption.yaml (hunk 2 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/age_exemption.yaml:10]: You are right, after the age of 65 canadians have the age amount tax credit but are still subject to income taxes.

Comment (MaxGhenis) [reply policyengine_canada/parameters/gov/age_exemption.yaml:10]: OK is `gov/age_exemption.yaml` the appropriate name for this parameter then?

File: policyengine_canada/variables/gov/income_tax.py [hunk 5]
```diff
@@ -12,7 +12,6 @@ class income_tax(Variable):
     def formula(person, period, parameters):
         income = person("total_individual_pre_tax_income", period)
         age = person("age", period)
-        gov = parameters(period).gov
+        gov = parameters(period).gov.cra.tax.income
         tax = gov.income_tax_schedule.calc(income)
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/income_tax.py:16]:
```suggestion
        return gov.income_tax_schedule.calc(income)
```

**Review by MaxGhenis — COMMENTED** [review]
Please similarly nest the tests and variables to `cra/tax/income`

File: policyengine_canada/variables/gov/income_tax.py (hunk 5 above)
Comment (PavelMakarchuk) [reply policyengine_canada/variables/gov/income_tax.py]: I have made the suggested adjustments.

**Comment by MaxGhenis** [comment]
Thanks - in the future please resolve conversations when you've addressed them, that helps confirm all outstanding issues are taken care of. Merging!

**Pull request merged**